tests/manual_test_coinbase.py

```diff
@@ -15,14 +15,19 @@
 # You should have received a copy of the GNU Affero General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-import base64
-import json
+import logging
 import sys
 
 from pyexchange.coinbase import CoinbaseApi
 from pymaker import Address, Wad
 
+logging.basicConfig(format='%(asctime)-15s %(levelname)-8s %(message)s', level=logging.INFO)
+logging.getLogger('urllib3').setLevel(logging.INFO)
+logging.getLogger('urllib3.connectionpool').setLevel(logging.INFO)
+logging.getLogger("requests").setLevel(logging.INFO)
+logging.getLogger('requests.packages.urllib3.connectionpool').setLevel(logging.INFO)
 
+# ctor params: api_server: str, api_key: str, secret_key: str, password: str
```

Comment:
not quite sure what `ctor` is referring to?

Reply:
Short for "constructor".  It's the parameters to pass.